_posts/2018-01-02-bbc-newshack.html

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title:  "Our explorations at BBC newsHack"
+date:   2018-01-02 09:30:30 +0100
+categories: Hackathon
+teaser: Hackathon
+excerpt: >
+  The FT explored conversational interfaces at BBC newsHack
+---
+<p>Back in October, the BBC organised a hackathon as part of their newsHack series. This time, they focused on Conversational User Interfaces (CUI) and, for the occasion, invited other news organisations to take part. The FT  joined in with a pluri-disciplinary team (editorial, UX design, software engineering).</p>
+<p>At the end of the 2-day event, we had a prototype ready for FT Conversation, an application running on Google Home, with our users' comments at its heart.</p>
+
+<h2>Ideation</h2>
+<p>After hearing about various tools and CUI challenges, we set out to find what would benefit the FT in that area. We were able to group our ideas in three over-arching themes: </p>
+<p>Discovery: Why not take advantage of our FT Weekend recipes and offer a cooking companion? Alternately, we could provide restaurant reviews, or even exhibition or events suggestions</p>
+<p>Navigating data (through audio): How can we envisage audio charts? This task felt too titanesque to undertake in a couple of days.</p>
+<p>As we went on, it became obvious that most suggestions were revolving aroung people's interactions; with various ideas around fact-checking, offering context, feedback and comments. We settled on exploring a few of those ideas, acting together in a voice assistant enable user conversations via our Comments section.</p>
```

Comment:
"acting together in a voice assistant enable user conversations via our Comments section"
doesn't parse